src/httpserver.cpp

```diff
@@ -410,10 +410,13 @@ bool InitHTTPServer()
 #if LIBEVENT_VERSION_NUMBER >= 0x02010100
     // If -debug=libevent, set full libevent debugging.
     // Otherwise, disable all libevent debugging.
-    if (LogAcceptCategory(LIBEVENT))
-        event_enable_debug_logging(EVENT_DBG_ALL);
-    else
-        event_enable_debug_logging(EVENT_DBG_NONE);
+    {
+        using namespace Logging;
+        if (LogAcceptCategory(LIBEVENT))
```

Comment:
Stylistically, it would be better to qualify the parameter rather than "using namespace" for a single item.  What I mean is its better to do:
LogAcceptCategory(Logging::LIBEVENT)

For background, the logging system hides the log categories under a namespace so that they can be both qualified and succinct.  We then employ a macro trick in the definition of LOG to effectively promote the namespace to the LOG parameters...  But for any other API you need to use the namespace.

Reply:
fair enough. going to change according to what you just said.